src/main/Restaurant.java

```diff
@@ -14,4 +24,28 @@ public float getCash() {
     public void setCash(float cash) {
         this.cash += cash;
     }
+    //method to add an ingredient into the list of ingredients
+    public void addIngredient(Ingredient ingredient){
+        this.ingredients.add(ingredient);
+    }
+
+    //getter to obtain the list of ingredients
+    public List<Ingredient> getIngredients() {
+        return this.ingredients;
+    }
+
+    //method to view the list of ingredients. Note that the way to view it it's according to
+    //the toString method implemented in the Ingredient class
+    public void showIngredients(List<Ingredient> ingredientList){
```

Comment:
Hey! I think this method `showIngredients` is not needed from the current scope of the ticket, so please go ahead a remove it